### PR TITLE
Wesnoth codesigning script

### PR DESCRIPTION
**Pull request by singalen**
Automates a part of app packaging and signing with Wesnoth, Inc certs

### CODE REVIEW

File: projectfiles/Xcode/wesnoth_codesign.sh [hunk 1]
```diff
@@ -0,0 +1,23 @@
+VERSION=1.13.12
+IN_FILENAME="Wesnoth_${VERSION} rw.dmg"
+OUT_FILENAME="Wesnoth_${VERSION}.dmg"
+VOLUME="/Volumes/Wesnoth ${VERSION}/"
+CODESIGN_IDENTITY='Wesnoth, Inc'
+
+set -euo pipefail
```
Comment (soliton-) [inline projectfiles/Xcode/wesnoth_codesign.sh]: -o pipefail is a bash feature. Either make it a bash script or just remove. There isn't any pipeline in this script anyway.

Comment (singalen) [reply projectfiles/Xcode/wesnoth_codesign.sh]: done.

**Comment by CelticMinstrel** [comment]
So with this, can we go on the Mac App Store?

**Comment by singalen** [comment]
We can do it even without this :)
I'm not really an Apple dev, but we have @hrubymar10 to chime in.

**Comment by singalen** [comment]
Didn't work in the end. Obsoleted by @hrubymar10 contribution.

**Pull request closed without merging**